src/css/components/_c-cta-get-started.scss

```diff
@@ -15,6 +15,10 @@
 		bottom: 2rem;
 	text-align: center;
 
+	@include mappy-bp(lap-large) {
+		bottom: 2.25rem;
```

Comment:
This is for positioning the CTA link to appear as close as possible to positioning in the current deployment.